project/Dependencies.scala

```diff
@@ -600,15 +600,22 @@ object Dependencies {
     akkaActor,
     akkaProtobuf,
     akkaSlf4j,
+    playJson,
     scalaXml,
     akkaHttpCore,
     akkaHttpRouteDsl,
     akkaHttpSprayJson,
     akkaParsing
   )
 
-  val `akka-management-javadsl` = libraryDependencies ++= Seq.empty[ModuleID]
-  val `akka-management-scaladsl` = libraryDependencies ++= Seq.empty[ModuleID]
+  val `akka-management-javadsl` = libraryDependencies ++= Seq(
+    // Upgrades needed to match whitelist
+    playJson
+  )
+  val `akka-management-scaladsl` = libraryDependencies ++= Seq(
+    // Upgrades needed to match whitelist
+    playJson
+  )
```

Comment:
I'm wondering why is this needed? AFAIK, the akka management module doesn't depend on play-json, also not transitively.

Reply:
I don't know why, but without this line, the whitelist flag was raised for akka-management-scaladsl..

Reply:
I will check it out. I just want to understand why this is happening.

Reply:
The reason is that `akka-management-{javadsl,scaladsl}` depends on `akka-management-core` that requires Play. Because Play is still in 2.7.0, we need to update the whilelist. It was enough to update it in `akka-management-core` tough. 

Anyway, none of this will be needed when we update it to Play 2.7.1.